packages/oc-s3-storage-adapter/index.js

```diff
@@ -27,25 +27,52 @@ module.exports = function(conf) {
     return true;
   };
 
+  // Defaults
+  const bucket = conf.bucket ? conf.bucket : '';
+  const sslEnabled = conf.sslEnabled === false ? false : true;
+  const s3ForcePathStyle = conf.s3ForcePathStyle ? true : false;
+  const signatureVersion = conf.signatureVersion
+    ? conf.signatureVersion
+    : AWS.Config.signatureVersion;
   const httpOptions = { timeout: conf.timeout || 10000 };
   if (conf.agentProxy) {
     httpOptions.agent = conf.agentProxy;
   }
 
-  AWS.config.update({
```

Comment:
Is there any further implication in moving from `config.update` to `Config` constructor?

Reply:
It seems like creating an instance:

```
AWS.Config({sslEnabled: false})
console.log(aws);
```

gives the same result  as:

```
AWS.config.update({sslEnabled:false});
console.log(AWS.config);
```

If you used update before then perhaps we should stick with that.

Reply:
I think is fine either way. I was just wondering if you were aware of any difference.